### PR TITLE
Update recover_test.go Copyright Year to 2022

### PR DESCRIPTION
**Pull request by umnooob**
update recover_test.go copyright year information to 2022

### Ⅰ. Describe what this PR does
Update pkg/csi/recover/recover_test.go Copyright Year to 2022

### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Comment by cheyang** [comment]
@umnooob please fix DCO issue by following 
```
1. Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
2. In your local branch, run: git rebase HEAD~1 --signoff
3. Force push your changes to overwrite the branch: git push --force-with-lease origin master
```

File: pkg/csi/recover/recover_test.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Fluid Authors.
+Copyright 2023 The Fluid Authors.
```
Comment (cheyang) [inline pkg/csi/recover/recover_test.go:2]: This source file is created in 2022, so the copyright is since 2022 not 2023.

**Comment by umnooob** [comment]
> This source file is created in 2022. Please update it from 2021 to 2022. Thanks.

updated, Thanks!

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**